examples/cli.py

```diff
@@ -141,6 +149,15 @@ def Main() -> None:
                 ('--disk_name', 'Name of the disk to copy. If None, the boot '
                                 'disk of the instance will be copied.', None)
             ])
+  AddParser('gcp', gcp_subparsers, 'startvm', 'Start a forensic analysis VM.',
+            args=[
+                ('instance_name', 'Name of the GCE instance to create.',
```

Comment:
Similar as above

Reply:
That's a good idea.

For GCP that's easy to do as it requires a list of disk names.
For AWS it needs a tuple (diskname,mountpoint) which would get nasty on the CLI or I take the list of disk names and loop through the whole alphabet for device mount points (/dev/sd[a-z]) depending on how many disks the user wants to mount.

wdyt?

Reply:
Ah, yes, I had the same issue when writing a recipe for AWS in dftimewolf. 
I ended up doing what you describe (with a limited alphabet though, to match AWS requirements), see https://github.com/log2timeline/dftimewolf/blob/10573673319032bf08311938ae834ee329ec576d/dftimewolf/lib/collectors/aws.py#L233 (still a WIP with things to address, but adding for context).

Reply:
okay, cool. probably not going to something with a pre-populated list, popping and checking. if Amazon supports f-p it means we can connect 11 volumes at a maximum. easier to check on that

Reply:
kept the fix as simple as possible in the cli code.

Reply:
This is great , thanks!